src/app/models/entities/userProfile.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { User } from "@repositories/user.repository";
+import {
+  Column,
+  Entity,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from "typeorm";
+
+@Entity({ name: "user_profile" })
```

Comment:
[Suggestion] Make the table name plural? 
```suggestion
@Entity({ name: "user_profiles" })
```